cpuinfo_test.go

```diff
@@ -232,6 +232,70 @@ processor	: 1
 hart		: 1
 isa		: rv64imafdcsu
 mmu		: sv48
+`
+
+	cpuinfoLoong64 = `
```

Comment:
I think placement of this block should be alphabetic as well. Which means putting it somewhere above.

Reply:
Other architectures are added later